src/makeRobotAccountant.js

```diff
@@ -4,9 +4,22 @@
  *
  * @return {function}
  */
+const ERROR_MESSAGE = 'Bzzz... Error!';
 
 function makeRobotAccountant() {
-  // write code here
+  let callsCounter = 0;
+
+  return (a) => {
+    return (b) => {
+      callsCounter++;
+
+      if (callsCounter < 3 || callsCounter % 2 !== 0) {
+        return a + b;
+      }
+
+      return ERROR_MESSAGE;
```

Comment:
Everything is well done. It seems that according to the condition, they did not ask to display an error. Well, maybe it was better to do it through throw new Error("ERROR_MESSAGE");